src/integTest/groovy/net/wasdev/wlp/gradle/plugins/TestCreateWithConfigDir.groovy

```diff
@@ -0,0 +1,39 @@
+package net.wasdev.wlp.gradle.plugins;
+
+import java.io.File
+
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.junit.Test
+
+public class TestCreateWithConfigDir extends AbstractIntegrationTest{
+    static File sourceDir = new File("build/resources/integrationTest/server-config")
+    static File testBuildDir = new File(integTestDir, "/test-create-with-config-dir")
+    static String buildFilename = "testCreateLibertyConfigDir.gradle"
+
+    @BeforeClass
+    public static void setup() {
+        createDir(testBuildDir)
+        if(test_mode == "offline"){
+            WLP_DIR.replace("\\","/")
+        }
+        createTestProject(testBuildDir, sourceDir, buildFilename)
+        renameBuildFile(buildFilename, testBuildDir)
+    }
+    
+    @Test
+    public void test_create_with_configDir() {
+
+        runTasks(testBuildDir, 'libertyCreate')
+
+        def bootstrapFile = new File("build/testBuilds/test-create-with-config-dir/build/wlp/usr/servers/LibertyProjectServer/bootstrap.properties")
+        def jvmOptionsFile = new File("build/testBuilds/test-create-with-config-dir/build/wlp/usr/servers/LibertyProjectServer/jvm.options")
+        def configFile = new File("build/testBuilds/test-create-with-config-dir/build/wlp/usr/servers/LibertyProjectServer/server.xml")
+        def serverEnvFile = new File("build/testBuilds/test-create-with-config-dir/build/wlp/usr/servers/LibertyProjectServer/server.env")
+
+        assert serverEnvFile.exists() : "file not found"
```

Comment:
if this fails, does the message say more than just "file not found"?  Does it say which file is not found?  If not add those to these 4 messages.